app/assets/stylesheets/govuk_publishing_components/components/_contents-list.scss

```diff
@@ -97,6 +97,12 @@
   }
 }
 
+.gem-c-contents-list--alternate-title {
+  .gem-c-contents-list__title {
+    margin-bottom: govuk-spacing(3);
```

Comment:
What's this spacing for? Is it required for the non-dashed style? I don't think it belongs with this option - I wouldn't expect passing a custom title to also alter the appearance of the component.

Reply:
Hmm, yeah this is a bit of a weird one because the extra space is required for the landing page design: it's to match the spacing of the `alternate-active` option which has larger line spacing than other contents-list variants:

![image](https://github.com/user-attachments/assets/0e8b6185-6ae5-4ff0-a684-b8d83bc2f582)

So the only time we need the extra space is if both a custom title is set AND the alternate-active option is used. Is it OK then to have a combined selector to handle this, eg: 

`.gem-c-contents-list--alternate-active.gem-c-contents-list--alternate-title`

Reply:
Let's not make it overly complicated. If this extra spacing is needed for the landing page design, then let's include it with that variant.

Reply:
Looks like this is still part of the custom title variant, can it be part of the alternative style variant instead?

Reply:
Deleted from the title variant.